website/docs/docs/deploy/deploy-jobs.md

```diff
@@ -67,9 +67,9 @@ To set your job's schedule, use the **Run on schedule** option to choose specifi
 
 Under **Timing**, you can either use customizable hours for jobs that need to run frequently throughout the day or exact intervals for jobs that need to run at specific times:
 
-- **Exact intervals** &mdash; Use this option to set how often your job runs, in hours. Enter a number between 1 and 23 to represent the interval between job runs. For example, if you set it to **Every 2 hours**, the job will run every 2 hours from midnight UTC. This option is useful if you need to run jobs multiple times per day at regular intervals.
+- **Intervals** &mdash; Use this option to set how often your job runs, in hours. Enter a number between 1 and 23 to represent the interval between job runs. For example, if you set it to **Every 2 hours**, the job will run every 2 hours from midnight UTC. This option is useful if you need to run jobs multiple times per day at regular intervals.
 
-- **Hours of the day** &mdash; Use this option to set specific times when your job should run. You can enter a comma-separated list of hours (in UTC) when you want the job to run. For example, if you set it to `0,12,23,` the job will run at midnight, noon, and 11 PM UTC. This option is useful if you want your jobs to run at specific times of day and don't need them to run more frequently than once a day.
+- **Specific hours** &mdash; Use this option to set specific times when your job should run. You can enter a comma-separated list of hours (in UTC) when you want the job to run. For example, if you set it to `0,12,23,` the job will run at midnight, noon, and 11 PM UTC. This option is useful if you want your jobs to run at specific times of day and don't need them to run more frequently than once a day.
```

Comment:
```suggestion
- **Specific hours** &mdash; Use this option to set specific times when your job should run. You can enter a comma-separated list of hours (in UTC) when you want the job to run. For example, if you set it to `0,12,23,` the job will run at midnight, noon, and 11 PM UTC (plus account-level jitter). This option is useful if you want your jobs to run at specific times of day and don't need them to run more frequently than once a day.

#### About account-level jitter

Jobs will always begin inside of the specified hour, but will not necessarily begin at 0 minutes past the hour. Each account has a fixed offset period which applies to all scheduled jobs in that account. If your account's offset period is 4 minutes and you have a job scheduled to run every 2 hours, it will run at 00:04 UTC, 02:04 UTC, 04:04 UTC and so on.
```

Reply:
I would do this as an example like the comment above showing the exact time this would run for these specific hours. Then I'd remove the account level jitter definition as it introduces a new concept/feature that adds value to us rather than to customers. 

So this could change to something like:
**Specific hours** &mdash; Use this option to set specific times when your job should run. You can enter a comma-separated list of hours (in UTC) when you want the job to run. For example, if you set it to `0,12,23,` the job will run at midnight, noon, and 11 PM UTC. Job runs will always be consistent between both hours and days, so if your job runs at 00:05, 12:05, and 23:05 UTC, it will run at these same hours each day. This option is useful if you want your jobs to run at specific times of day and don't need them to run more frequently than once a day.